docs/variants/msi_z690/building.md

```diff
@@ -0,0 +1,50 @@
+# Dasharo compatible with MSI PRO Z690-A WIFI DDR4 - building manual
+
+## Intro
+
+This documents describes the procedure for compiling Dasharo firmware
+compatible with MSI PRO Z690-A WIFI DDR4.
+
+## Requirements
+
+* `Ubuntu 20.04 LTS` as a host OS
+* Internet connection
+* Docker installed
+    - follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    - follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* Git, wget, unzip installed
+
+```bash
+sudo apt install git  unzip wget
+```
+
+* UEFIExtract installed
```

Comment:
@miczyg1 maybe one sentence why we need this tool?

Reply:
@IgorBagnucki